app/services/copy_phone_configurations.rb

```diff
@@ -0,0 +1,28 @@
+class CopyPhoneConfigurations
```

Comment:
What do you think about naming this `CopyPhonesToPhoneConfigurations` for more accuracy?